wiki/CLI-language.md

```diff
@@ -428,6 +428,14 @@ By adding the filter tag=[tag_slug] to the ls, we can obtain the children that h
 ls [path] tag=[tag_slug]
 ```
 
+#### Complex filters
+
+Complex filters can be added to the ls command with the flag `-f`, composing complex boolean expressions with the operators `=`, `!=`, `<`, `<=`, `>`, `>=`, `&` and `|`.
+
+```
```

Comment:
adding more examples would be nice, as well as mentioning the () in the text. And what about the get?

Reply:
I've rewritten this section of the Wiki to better explain the functioning of the complex filter, and added a section about it to the `get` command.